Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -749,6 +737,22 @@ private void AddLinks()
                 }
             }
 
+            // Aligns each rotation handle with the Y axis along the edge and the X and Z axis pointing
```

Comment:
It would be useful to describe alignment of the prefabs in a tooltip for the rotation prefab, since some people will not have easy access to the raw code and may not see this comment.

Reply:
Done.